Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -59,7 +59,10 @@ There are usually two release engineers working at any given time. It goes witho
 1. Any hotfix should target the `develop` branch first. 
 1. These PRs need to be reviewed by the relevant squad or platform QA (Bibin Paul) & one of the release engineers assigned to the release. This is to ensure visibility of changes being requested to be made, and to ensure the correct builds are available for validation. The exception being that if an issue is not reproducible on `develop` then we can merge directly with `release`. Still, it should be fully understandable why it is not reproducible to help ensure there is no regression for the next release.
     * Bear in mind that two approvals from other engineers assigned by Pull Assigners is not enough in this particular case.
-    * After merging, we'll need to cherry-pick the commit via a PR to the release branch. Doing this will ensure that both branches are up to date and reduces the risk of conflicts when merging the release branch back to develop. 
+    * After merging, we'll need to merge these changes to the release branch. This can be done either via cherry-picking relevant commits or rebasing PR branch on release branch. _This should be done by author of the change and PR should be reviewd by release engineer as well_. Doing this will ensure that both branches are up to date and reduces the risk of conflicts when merging the release branch back to develop.
```

Comment:
```suggestion
    * After merging, we'll need to merge these changes to the release branch. This can be done either via cherry-picking the relevant commits or rebasing the PR branch on the release branch. _This should be done by the author of the change and the PR should be reviewed by the release engineer as well_. Doing this will ensure that both branches are up to date and reduces the risk of conflicts when merging the release branch back to develop.
```